sdk/ai/azure-ai-projects/tests/agents/test_agents_crud_async.py

```diff
@@ -14,7 +14,7 @@
 class TestAgentCrudAsync(TestBase):
 
     @servicePreparer()
-    @recorded_by_proxy_async
+    @recorded_by_proxy_async()
```

Comment:
Hm... we currently use the decorator without parentheses throughout the repo; will this require each instance to be updated?